tspng/extraction.py

```diff
@@ -6,6 +6,16 @@
 import os
 
 def extract_from_file(path, mime_type=MIME_TYPE):
+    '''
+    Returns the metadata from a TSPNG file as a dictionary.
+
+        Parameters:
+                path (str): Path to a file as a string
+                mime_type (str): Optional; Media type of file
```

Comment:
I would add something about it defaulting to the `MIME_TYPE` constant and the value being `application/vnd.theiascope.io+json`

Reply:
Added something for now